### ISSUE TITLE
Uploading to a different drive seems to fail.

### ISSUE THREAD

**Issue by fonym** [robinrodricks/FluentFTP#153]
Hello,

Uploading a file where the path includes a drive letter doesn't work.
When connecting to the FTP server it's working directory is "C:/" but my files locations are on "F:/" if my upload path includes the drive letter it fails during the OpenPassiveDataStream method because it tries to open the stream at "C:/F:/" so I thought to change the working directory first to "F:/" and use only the file name instead of full path but it still fails because now OpenPassiveDataStream tries "F:/F:/"
```
Response: 250 Changed directory to "F:"

UploadFile("C:\Temp\temp123.csv", "temp.csv", Overwrite, False, Retry, Delete, Throw)

FileExists("temp.csv")

GetWorkingDirectory()
Command:  PWD
Response: 257 Current directory is "F:"

GetNameListing("F:")

GetWorkingDirectory()
Command:  PWD
Response: 257 Current directory is "F:"
Command:  TYPE I
Response: 200 Type set to I, binary mode

OpenPassiveDataStream(AutoPassive, "NLST F:/F:", 0)
Command:  EPSV
Response: 500 Command not recognized
```

**Comment by fonym** [comment]
Hi,

I have overcome the issue by changing line 1446 of FtpClient_connection.cs from

} else if (!path.StartsWith("/")) {
to
} else if (!path.StartsWith("/") && path.Substring(1,1) != ":") {

No doubt there are more elegant solutions but it gets me out of a hole for now.

**Comment by robinrodricks** [comment]
Thanks for this fix. I've implemented this in 19.2.0.
